source/fundamentals/promises.txt

```diff
@@ -0,0 +1,242 @@
+======================
+Promises and Callbacks
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecols
+
+Overview
+--------
+
+The NodeJS driver uses the asynchronous Javascript API to communicate with
```

Comment:
Why do we refer to it as `NodeJS` when the official Node site refers to it as `Node.js`?

Not having the correct naming is like saying `Javascript` instead of `JavaScript`. Unless there is a specific reason, I think we lose credibility with developers.

Reply:
I think this was a team decision made some time ago. Perhaps @terakilobyte or @mbroadst can weigh in on how we want to represent it and then we can change it across all the driver documentation.

Reply:
:wave: It should be `Node.js`

Reply:
Sounds good, created a ticket to do this across all the guidance docs:
https://jira.mongodb.org/browse/DOCSP-10080